src/Component/Authentication/Login.jsx

```diff
@@ -0,0 +1,379 @@
+                position: "absolute",
+                right: 8,
+                top: 8,
+                color: "white",
+              }}
+            >
+              <CloseIcon />
+            </IconButton>
+          </DialogTitle>
+          <DialogContent
+            sx={{
+              color: "white",
+              display: "flex",
+              flexWrap: "wrap",
+              flexDirection: "column",
+              justifyContent: "center",
+              alignItems: "center",
+              gap: 2,
+              width: 400,
+              height: 1,
+            }}
+          >
+            <Typography color="white" textAlign="center">
+              {" "}
+              Not a member yet?{"    "}
+              <Link href="/signup" variant="body2" style={{ color: "white" }}>
+                Sign Up
+              </Link>
+            </Typography>
+            <TextField
+              sx={{
+                border: "white",
+                backgroundColor: "white",
+                borderRadius: "1rem",
+                "& .MuiInputBase-root": {
+                  "&:before": {
+                    borderBottom: "none",
+                  },
+                  "&:hover:before": {
+                    borderBottom: "none",
+                  },
+                  "&:after": {
+                    borderBottom: "none",
+                  },
+                },
+              }}
+              id="email"
+              error={emailError}
+              label="Email"
+              value={emailInput}
+              onChange={(event) => setEmailInput(event.target.value)}
+              onBlur={handleLoginEmail}
+              variant="filled"
+              fullWidth
+              size="small"
+            />
+            <FormControl
+              sx={{
+                border: "white",
+                backgroundColor: "white",
+                borderRadius: "1rem",
+                "& .MuiInputBase-root": {
+                  "&:before": {
+                    borderBottom: "none",
+                  },
+                  "&:hover:before": {
+                    borderBottom: "none",
+                  },
+                  "&:after": {
+                    borderBottom: "none",
+                  },
+                },
+              }}
+              variant="filled"
+              fullWidth
+              size="small"
+            >
+              <InputLabel
+                error={passwordError}
+                htmlFor="filled-adornment-password"
+              >
+                Password
+              </InputLabel>
+              <Input
+                error={passwordError}
+                type={showPassword ? "text" : "password"}
+                value={passwordInput}
+                onChange={(event) => setPasswordInput(event.target.value)}
+                onBlur={handleLoginPassword}
+                endAdornment={
+                  <InputAdornment position="end">
+                    <IconButton
```

Comment:
Could we align the icon? 
![Screen Shot 2024-12-11 at 7 05 00 PM](https://github.com/user-attachments/assets/c2c47361-0fd6-4ddc-9f2a-3561de09d45a)

Reply:
Aligned the icon